src/main.rs

```diff
@@ -90,6 +93,19 @@ fn main() {
                      clearing disk caches, for example.",
                 ),
         )
+        .arg(
+            Arg::with_name("style")
+                .long("style")
+                .short("s")
+                .takes_value(true)
+                .value_name("OPT")
+                .possible_values(&["auto", "basic", "full"])
+                .help(
+                    "Set output style type. If set to 'basic', all colors and special \
```

Comment:
We could maybe add `( default: auto)`.

Reply:
It affected the output when hyperfine was called with no arguments. When I get home from work today I'll update this comment.

Reply:
Actually I misunderstood what you meant by default. I was talking about when I added it as a call on Arg (default_value("auto")). But yes, I can add (default: auto) to the help output.